### ISSUE TITLE
Various Artist Query on Gazelle Music Trackers

### ISSUE THREAD

**Issue by ItsME6969** [Jackett/Jackett#12805]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using HTTP Client: HttpWebClient2
Using FlareSolverr: Yes
Using proxy: Disabled
App config/log directory: /config/Jackett
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
Running in Docker: Yes (image build: unknown)
File /etc/issue: Ubuntu 18.04.6 LTS \n \l
Jackett variant: CoreLinuxAmdx64	
OS version: Unix 4.4.180.0 (64bit OS) (64bit process)
Environment version: 6.0.1 (/app/Jackett/)
Jackett v0.20.285
```

### Description

Bug report/Feature request
Lidarr queries VA Compilations as VA - Album Name but searching that wont yield any results on gazelle music trackers. That query works well for Scene releases but not on P2P releases. So maybe filter out the VA during search and just search the album title or make it an optional choice when configuring the trackers.

Searches
1) VA - Album Title

![0](https://user-images.githubusercontent.com/89707006/148904527-7e49654e-76a3-450a-9eda-91d666582812.jpg)

2) Album Title

![1](https://user-images.githubusercontent.com/89707006/148904662-9ee116ba-fe5f-49ad-8ee8-adffce3e3121.jpg)

3) Various Artists - Album Title

![2](https://user-images.githubusercontent.com/89707006/148904703-6068797c-6c22-4326-bd9e-6f34cbc05a75.jpg)

As you can see The VA needs to be dropped. Searching using Various Artists also no joy even though that is in the release name. 

Looks like there is an issue that is Open in Lidarr but looks to be stale.

So basically an if ( Query contains Various Artists) then remove that.

https://github.com/Lidarr/Lidarr/issues/1180

### Logged Error Messages

Torznab search in Orpheus for VA Some Album => Found 0 releases

**Comment by ilike2burnthing** [comment]
The Gazelle abstract doesn't search by artist/album from a Jackett dashboard search.

Dashboard:
`searchstr=Various+Artists+-+Album+Title`

API:
`artistname=Various+Artists&groupname=Album+Title`

You'll need to try your 3 searches again with the API, e.g.:

```text
http://127.0.0.1:9117/api/v2.0/indexers/orpheus/results/torznab/api?t=music&cat=3000&extended=1&apikey=APIKEY&offset=0&limit=100&q=&artist=Various%20Artists&album=Album%20Title
```

In saying that though, it's odd that no results are given when doing a normal search with the format `Various Artists - Album Title`. Can you confirm that you're able to successfully search like this on Orpheus' site itself?

**Comment by ItsME6969** [comment]
@ilike2burnthing 
ON THE SITE

https://orpheus.network/torrents.php?searchstr=VA+-+Album+Name
`No results`
https://orpheus.network/torrents.php?searchstr=Various+Artists+-+Album+Name
`No results` --> That will never work with VA releases
https://orpheus.network/torrents.php?searchstr=Album+Name
`Works`

URLs

https://orpheus.network/torrents.php?artistname=Various+Artists&groupname=Album+Name
`Works`
https://orpheus.network/torrents.php?artistname=VA&groupname=Album+Name
`Works`

The URLs in that format work and take you **straight** to the release group/torrent ID (**if it exists**), even with VA
otherwise, it just does a basic search and returns everything that matches the groupname searched neglecting the `VA|Various Artists`--->basically it changes the search to `https://orpheus.network/torrents.php?action=advanced&groupname=Album+Name`

VIA POSTMAN

http://my.IP.Address/api/v2.0/indexers/orpheus/results/torznab/api?t=music&cat=3000&extended=1&apikey={{JackettAPI}}&offset=0&limit=100&q=&artist=Various+Artists&album=Album+Title
`No Results`
http://my.IP.Address/api/v2.0/indexers/orpheus/results/torznab/api?t=music&cat=3000&extended=1&apikey={{JackettAPI}}&offset=0&limit=100&q=&artist=VA&album=Album+Title
`No Results`
http://my.IP.Address/api/v2.0/indexers/orpheus/results/torznab/api?t=music&cat=3000&extended=1&apikey={{JackettAPI}}&offset=0&limit=100&q=&artist=&album=Album+Title
`Works`

So basically looks like an easy fix.
Jackett looks at the Query from Lidarr...
If Artist Name is VA|Various Artists filter it out and Just Search the Album Title.

To mitigate and reduce the False Positives, Just add Year to the Query and everything works perfectly like so

http://my.IP.Address/api/v2.0/indexers/orpheus/results/torznab/api?t=music&cat=3000&extended=1&apikey={{JackettAPI}}&offset=0&limit=100&q=&album=`Album+Title&year=year`
`Works Perfectly`

**Comment by ItsME6969** [comment]
![Screenshot 2022-01-11 153917 - Copy](https://user-images.githubusercontent.com/89707006/149043579-46dd4af1-d095-4386-8fe4-46d31aca4ee7.jpg)

**Comment by ilike2burnthing** [comment]
It's confusing me that `artistname=Various+Artists&groupname=Album+Name` (and VA) are working on the site, but `artist=Various+Artists&album=Album+Title` (and VA) are failing for Jackett, as they should be basically the same.

Check again with https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&artistname=Various+Artists&groupname=Album+Title&filter_cat[1]=1&filter_cat[4]=1 (and VA) as that's what Jackett actually uses.

If somehow that works for you in browser, then please provide an enhanced log of the failed Jackett API search for `Various Artists` - https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket

**Comment by ItsME6969** [comment]
@ilike2burnthing 
https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&artistname=Various+Artists&groupname=Album+Title&filter_cat%5B1%5D=1&filter_cat%5B4%5D=1
does not work

I guess I put too much info in that comment. Basically when you do a basic search on the actual tracker it uses `searchstr` and the url will look like below and you cannot find any album on the tracker when your search term includes VA or Various Artists in it simply because, there is no artist called Various artist but the URLs will be as below.

https://orpheus.network/torrents.php?searchstr=VA+-+Album+Name
`No results`
https://orpheus.network/torrents.php?searchstr=Various+Artists+-+Album+Name
`No results` 
https://orpheus.network/torrents.php?searchstr=Album+Name
`Works`

The 2nd set of links was me just modifying the URLs, NOT what you see when you do a search.

**Comment by ilike2burnthing** [comment]
Ok, so we know:
https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&artistname=Various+Artists&groupname=Album+Title&filter_cat[1]=1&filter_cat[4]=1
and `VA` don't work.

I'll need you to check:
https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&artistname=Michael+Jackson&groupname=Thriller&filter_cat[1]=1&filter_cat[4]=1
and:
https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&searchstr=Various+Artists+-+Album+Name&filter_cat[1]=1&filter_cat[4]=1
and:
https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&searchstr=Various+Artists+Album+Name&filter_cat[1]=1&filter_cat[4]=1
and the last two with `VA` as well.

Then we can determine what needs stripped when.

**Comment by ItsME6969** [comment]
Cool we'll just wait I'll update when its is fixed.

**Comment by phoenixxx-1** [comment]
I'm seeing this same Various Artists search issue.  The ability to add a series as an import list in the latest develop release is what has caused me to notice this.  I believe the comments already shown in this thread articulate the problem well;

Simply put, when performing a search for "Various Artists" against a gazelle tracker, "VA " should be dropped from the prefix of the search.

**Comment by ilike2burnthing** [comment]
A year on, has this been resolved?

**Comment by ItsME6969** [comment]
@ilike2burnthing unfortunately still hasn't been resolved no ETA either
